scripts/plots/global_ens/exevs_global_ens_headline_gefs_grid2grid_plots.sh

```diff
@@ -34,7 +34,7 @@ else
   last_year=$this_year
 fi 
 
-export PLOT_CONF=$PARMevs/metplus_config/global_ens/headline_grid2grid/plots
+export PLOT_CONF=$PARMevs/metplus_config/plots/global_ens/headline_grid2grid
```

Comment:
Nitpicking, but please update line 37 to use $STEP and $COMPONENT instead of plots and global_ens. Change to:
```
export PLOT_CONF=$PARMevs/metplus_config/$STEP/$COMPONENT/headline_grid2grid
```